styles.css

```diff
@@ -705,4 +705,96 @@ textarea::placeholder {
     letter-spacing: 0.03em;
     color: #396df2;
   }
+
+  .card-reverse {
+    flex-direction: row-reverse;
+  }
+
+  .about-me {
+    height: 700px;
+    padding-top: 100px;
+  }
+
+  .about-container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .about-container div {
+    width: 50%;
+  }
+
+  .about-title {
+    width: 100px;
+  }
+
+  .connect {
+    margin: 40px 0;
+  }
+
+  .about-stack {
+    display: flex;
+    flex-direction: column;
+    width: 40%;
+  }
+
+  .stack-container {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .stack-category {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+  }
+
```

Comment:
[OPTIONAL] To produce a good and nice looking website, kindly ensure that you follow all the design guidelines from the figma template to the letter. Check and fix the disparities on the project card including the margins and paddings :+1: 

![Screenshot from 2023-01-21 23-14-32](https://user-images.githubusercontent.com/65233860/213892931-82481faa-5ee2-4659-bd32-18a46b0a82e4.png)